server/parsec/asgi/administration.py

```diff
@@ -755,3 +760,36 @@ async def administration_organization_sequester_service_update_config(
         status_code=200,
         content={},
     )
+
+
+class AdministrationUsersOrganizations(BaseModel):
+    user_email: str
+
+
+@administration_router.get("/administration/users/organizations")
```

Comment:
Since this API is about getting back organization ID, it should be

```suggestion
@administration_router.get("/administration/organizations")
```
Used as `GET /administration/organization?human_email=john@example.com`

Reply:
However if we want to also provide information about the users with the given human handle in each organization, then we probably provide this route with a more specialized name, for instance:

```json5
$ curl https://parsec.cloud/administration/organizations_with_human?email=john@example.com
{
    organizations: [
      {
         organization_id: CoolOrg,
         users: [ // Ordered by created_on
            {
               user_id: b63f033a5a0042faa81be96c9e6f3ee7,
               created_on: "2000-01-02T00:00:00Z",
               current_profile: ADMINISTRATOR,
               frozen: false,
               revoked_on: None,
             },
            {
               user_id: b63f033a5a0042faa81be96c9e6f3ee7,
               created_on: "2000-01-01T00:00:00Z",
               current_profile: STANDARD,
               frozen: false,
               revoked_on: "2000-01-02T00:00:00Z",
             }
         ]
       }
    ]
}
```